Program.cs

```diff
@@ -0,0 +1,410 @@
+﻿using System;

```

Comment:
the order must to be applied.
first System....
after Microsoft...
then ThirdParty namespaces
and last another project namespaces.

please remember this, is very important.

Reply:
Solved